deployment/ccip/changeset/testhelpers/test_helpers.go

```diff
@@ -1682,26 +1680,6 @@ func NewMintTokenWithCustomSender(auth *bind.TransactOpts, sender *bind.Transact
 	return MintTokenInfo{auth: auth, sender: sender, tokens: tokens}
 }
 
-// ApproveToken approves the router to spend the given amount of tokens
-func ApproveToken(env cldf.Environment, src uint64, tokenAddress common.Address, routerAddress common.Address, amount *big.Int) error {
-	token, err := erc20.NewERC20(tokenAddress, env.Chains[src].Client)
-	if err != nil {
-		return err
-	}
-
-	tx, err := token.Approve(env.Chains[src].DeployerKey, routerAddress, amount)
-	if err != nil {
-		return err
-	}
-
-	_, err = env.Chains[src].Confirm(tx)
-	if err != nil {
-		return err
-	}
-
-	return nil
-}
-
```

Comment:
With the new refactoring, are you able to do this now? 👀 

```suggestion
// ApproveToken approves the router to spend the given amount of tokens	
func ApproveToken(env cldf.Environment, src uint64, tokenAddress common.Address, routerAddress common.Address, amount *big.Int) error {	
	return changeset.ApproveToken(env, src, tokenAddress, routerAddress, amount)
}	

```